lib/registries/addon/drafts/draft/metadata/styles.scss

```diff
@@ -0,0 +1,51 @@
+.ContentBackground {
```

Comment:
these styles are not necessary as they apply to `OsfLayout` and parent route (draft route) will take care of the layout.